.github/workflows/release-helm-chart.yml

```diff
@@ -0,0 +1,24 @@
+name: Release helm chart
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'helm/charts/**/*'
+
+jobs:
+  push-chart-to-gitlab:
+    runs-on: ubuntu-latest
```

Comment:
Is there a reason to use latest?  And not specific version? Its always better to use fixed version to prevent the pipeline from failing when the OS is upgraded.
Other question: Why not use debian minimal?

Reply:
Tbh, just copy paste from other place.

Reply:
I see we have ubuntu-latest for all of our actions, so let' s keep it.
We can discuss/change it for all in other ticket.